uspv/header_test.go

```diff
@@ -1,96 +0,0 @@
-package uspv
```

Comment:
Why was this deleted?

Reply:
this is now part of the python integration test framework

Reply:
But as an integration test, this unit test just checks to see if the USPV code can detect/process reorgs sensibly.  The integration test checks to see if lit functions reasonably when a reorg actually happens.

Reply:
Agreed, but `moreWork` is pretty useless unless you have a chain, hence its place in the python framework. Also, its easier to create blocks using bitcoind since you don't need to worry about nonces and stuff whereas if you craft it, you do have to.

Reply:
But then if lit starts breaking when reorgs happen we won't know if it's because someone refactored the uspv code and broke the something involving reorgs there or if someone wrote code higher up that doesn't sensibly handle reorging.  Sure the blocks we're making in the test aren't really sensible but the important part is that it can calculate which chain has more work, even if the chain is just a few blocks long.

Reply:
There's no cost to keeping this as it should still work, so let's just leave it in.

Reply:
Will restore.